docs/v2/11_web3wallet-mobile-and-desktop.md

```diff
@@ -172,39 +173,43 @@ public class Web3WalletSignVerify : MonoBehaviour
 
 ### Hosting Your Own Sign Page {#hosting-your-own-sign-page}
 
-You can find the Web3Wallet repo here [https://github.com/ChainSafe/game-web3wallet](https://github.com/ChainSafe/game-web3wallet) you can download and host this on git for free. This instantiates web3 in webpage to allow MetaMask to connect & make transactions. We'll guide you through this process below.
+You can find the Web3Wallet repo [here](https://github.com/ChainSafe/game-web3wallet). You can download and host this on git for free. This instantiates web3 in a webpage to allow MetaMask to connect & make transactions. We'll guide you through this process below.
 
-Step 1: Create a GitHub account [https://github.com/join](https://github.com/join)\
+Step 1: [Create a GitHub account](https://github.com/join).
 
-
-Step 2: Clone the web3wallet repo using GitHub Desktop
+Step 2: Clone the [Web3Wallet](https://github.com/ChainSafe/game-web3wallet) repo using GitHub Desktop. Press `Add`, `Clone Repository...`, and then paste in the following URL: `https://github.com/ChainSafe/game-web3wallet`.
 
 ![](<v2Assets/image (14).png>)
 
-Step 3: If it doesn't ask you automatically, in the top left, press add, then press clone repository and paste in the URL [https://github.com/ChainSafe/game-web3wallet](https://github.com/ChainSafe/game-web3wallet)
-
 ![](<v2Assets/image (12).png>)
 
-Step 4: GitHub desktop will ask you if you want to fork the repo. When prompted select "For My Own Purposes". This will create a fork that you can push to your own repository.
+Step 3: GitHub Desktop will ask you if you want to fork the repo. When prompted, select `For My Own Purposes`. This will create a fork, where you can push to your own repository.
```

Comment:
This assumption is that the user is using Github Desktop?

Reply:
We will have to rewrite this part in the refactor to be more inclusive, but this first pass-through is meant to be a touch-up and not anything structurally significant